README.md

```diff
@@ -4,19 +4,19 @@
 
 🔨 **Looking for contributors** 🔨 If you feel like getting involved, please get in contact!
 
-BreakTimer is a desktop application for managing and enforcing periodic breaks. BreakTimer is available for Windows, macOS and Linux.
+BreakTimer is a desktop application for managing and enforcing periodic breaks. BreakTimer is available for Windows, macOS, and Linux.
 
 BreakTimer allows you to customize:
 
-- How long your breaks are and how often you wish to have them
+- How long your breaks are and how often do you wish to have them
 - Whether to be reminded with a simple notification or a fullscreen break window
 - Working hours so you are only reminded when you want to be
 - The content of messages shown during breaks.
 - Whether to intelligently restart your break countdown when it detects that you have not been using the computer
 
 ## Installation
 
-- **Windows** - [BreakTimer.exe](https://github.com/tom-james-watson/breaktimer-app/releases/latest/download/BreakTimer.exe) (unsigned - you will receive a warning on install, press more info -> run anyway)
+- **Windows** - [BreakTimer.exe](https://github.com/tom-james-watson/breaktimer-app/releases/latest/download/BreakTimer.exe)
```

Comment:
```suggestion
- **Windows** - [BreakTimer.exe](https://github.com/tom-james-watson/breaktimer-app/releases/latest/download/BreakTimer.exe) (unsigned - you may receive a warning on install, press more info -> run anyway)
```

It's definitely still unsigned - we can change this to "may" if this is not always shown, though.